fala/templates/adviser/results.html

```diff
@@ -8,6 +8,22 @@
   <div class="govuk-grid-column-full">
     <div id="google_translate_element" class="govuk-!-margin-bottom-6 govuk-!-display-none-print"></div>
   </div>
+  {% if tailored_results %}
+    <div class="govuk-grid-column-full">
+      <form action="/single-category-search" method="GET" novalidate>
+        <input type="hidden" name="postcode" value="{{ form.postcode.value() }}">
+        <input type="hidden" name="name" value="{{ form.name.value() }}">
+        {% for value, label_text in form.categories.field.choices %}
+          {% if label_text in form|category_selection_list %}
+            <input type="hidden" name="categories" value="{{ value }}">
+          {% endif %}
+        {% endfor %}
+        <a href="/single-category-search?postcode={{ form.postcode.value() }}&name={{ form.name.value() }}{% for value, label_text in form.categories.field.choices %}{% if label_text in form|category_selection_list %}&categories={{ value }}{% endif %}{% endfor %}" class="govuk-back-link govuk-!-margin-bottom-5 govuk-!-display-none-print">
+          Back
+        </a>
+      </form>
+    </div>
+  {% endif %}
```

Comment:
@willc-work thinking about this again we could have less input options in the back link url. I don't think we actually need an organisation name or postcode to create a back link to the `single-category-search` page. All we need is the category. It'll get tricky when CLA pass us more than one category, but we'll deal with that on another ticket.

This is the change I would make. 

```
    <div class="govuk-grid-column-full">
      <form action="/single-category-search" method="GET" novalidate>
        {% for value, label_text in form.categories.field.choices %}
          {% if label_text in form|category_selection_list %}
            <input type="hidden" name="categories" value="{{ value }}">
          {% endif %}
        {% endfor %}
        <a href="/single-category-search?{% for value, label_text in form.categories.field.choices %}{% if label_text in form|category_selection_list %}categories={{ value }}{% endif %}{% endfor %}" class="govuk-back-link govuk-!-margin-bottom-5 govuk-!-display-none-print">
          Back
        </a>
      </form>
    </div>
```

Reply:
good call